modules/nw-chaining-multiple-interface-container.adoc

```diff
@@ -0,0 +1,173 @@
+// Module included in the following assemblies:
```

Comment:
I don't see the need to have a new section. It's clearer if you add to the previous section the content from line 14. I mean, just the example. No need to get into details of SRIOV

Reply:
I merged into the previous section as per your suggestion.